java/servlets/security/cookie-issecure-false.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: cookie-issecure-false
+  patterns:
+  - pattern-either:
+    - pattern: |
+        $COOKIE = new Cookie(...);
+        ...
+  - pattern-not-inside: |
+      $COOKIE = new Cookie(...);
+      ...
+      $COOKIE.setSecure(true);
+  message: |
+    Default session middleware settings: `setSecure` not set to true. 
+    This ensures that the cookie is sent only over HTTPS to prevent cross-site scripting attacks.
+  fix-regex:
+    regex: 'false'
```

Comment:
Should we be a little tighter on the bounds where `false` can occur? E.g. `\(false\)` -> `\(true\)`? My concern is that any `false` that occurs in the `pattern` via the ellipsis operator will be fixed, not necessarily just a `setSecure(false)`.

Reply:
good point -- I'll change this.